.github/workflows/build.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: |
           echo -e '\n\n' | pmbootstrap init || true
           cd ~/.local/var/pmbootstrap/cache_git/pmaports
-          git remote add sm7150 https://github.com/sm7150-mainline/pmaports.git
```

Comment:
Open the PR with the fix at https://github.com/sm7150-mainline/pmaports.git instead of using your repository

Reply:
Better yet, right here: https://gitlab.com/postmarketOS/pmaports